tests/at_functional_test/test/atclient_put_test.dart

```diff
@@ -2,171 +2,123 @@ import 'dart:io';
 import 'dart:typed_data';
 
 import 'package:at_client/at_client.dart';
-import 'package:at_commons/at_commons.dart';
+import 'package:at_functional_test/src/at_keys_intialializer.dart';
 import 'package:test/test.dart';
-
-import 'set_encryption_keys.dart';
 import 'test_utils.dart';
 
 /// The tests verify the put and get functionality where key is created using AtKey concrete
 /// class
 void main() {
-  test('put method - create a key sharing to other atSign', () async {
-    var atsign = '@alice🛠';
-    var preference = TestUtils.getPreference(atsign);
-    final atClientManager = await AtClientManager.getInstance()
-        .setCurrentAtSign(atsign, 'me', preference);
-    var atClient = atClientManager.atClient;
-    atClient.syncService.sync();
+  late AtClientManager atClientManager;
+  String atSign = '@alice🛠';
+
+  setUpAll(() async {
+    var preference = TestUtils.getPreference(atSign);
+    atClientManager = await AtClientManager.getInstance()
+        .setCurrentAtSign(atSign, 'wavi', preference);
     // To setup encryption keys
-    await setEncryptionKeys(atsign, preference);
+    await AtEncryptionKeysLoader.getInstance()
+        .setEncryptionKeys(atClientManager.atClient, atSign);
+  });
+
+  test('put method - create a key sharing to other atSign', () async {
     // phone.me@alice🛠
     var phoneKey = AtKey()
       ..key = 'phone'
       ..sharedWith = '@bob🛠';
     var value = '+1 100 200 300';
-    var putResult = await atClient.put(phoneKey, value);
+    var putResult = await atClientManager.atClient.put(phoneKey, value);
     expect(putResult, true);
-    var getResult = await atClient.get(phoneKey);
+    var getResult = await atClientManager.atClient.get(phoneKey);
```

Comment:
For tests like this, the sync is happening as a side effect. It would be great if we can prevent it so that tests will be faster. Options could be:

1. Write key to key store but not commit log
2. Test using unsynced keys
3. Futuristic: shouldSync can be an attribute on probably metadata where setting it to false will prevent a sync and it can be defaulted to true